frontend/packages/shared/src/components/TreeView/TreeViewItem/TreeViewItem.tsx

```diff
@@ -123,7 +123,7 @@ export const TreeViewItem = ({
       type='button'
       variant='tertiary'
     >
-      {label}
+      <div className={classes.label}>{label}</div>
```

Comment:
We should consider if we always want truncate/ellipsis on the TreeViewItem, but I cant see where we do not want it. 🙈